client/src/components/admin/problem.js

```diff
@@ -0,0 +1,167 @@
+import { Fragment } from 'preact'
+import withStyles from '../../components/jss'
+import { useState, useCallback } from 'preact/hooks'
+import Modal from '../../components/modal'
+
+import { updateChallenge, deleteChallenge } from '../../api/admin/challs'
+import { useToast } from '../../components/toast'
+
+const DeleteModal = withStyles({
+  modalBody: {
+    paddingTop: '0em !important' // reduce space between header and body
+  },
+  controls: {
+    display: 'flex',
+    justifyContent: 'center',
+    '& :first-child': {
+      marginLeft: '0em'
+    },
+    '& :last-child': {
```

Comment:
It would be a bit more clean if we replaced these with individual selectors for the cancel and delete challenge buttons, in case we ever add more buttons.

Reply:
This is already set up to allow for more buttons, the selectors are for targeting whichever button is first and last to remove extra margin from the outside of the buttons; it isn't targeting the cancel and delete buttons specifically, but instead their positions.